Altis_Life.Altis/core/actions/fn_impoundAction.sqf

```diff
@@ -19,7 +19,8 @@ _vehicleName = FETCH_CONFIG2(getText,"CfgVehicles",(typeOf _vehicle),"displayNam
 _price = M_CONFIG(getNumber,"LifeCfgVehicles",(typeOf _vehicle),"price");
 [0,"STR_NOTF_BeingImpounded",true,[((_vehicleData select 0) select 1),_vehicleName]] remoteExecCall ["life_fnc_broadcast",RCLIENT];
 life_action_inUse = true;
-
+_uid =  (_vehicleData select 0) select 0;
+_owner = (playableUnits select {getPlayerUID _x == _uid}param[0, objNull]);
```

Comment:
Should be able to use [`isEqualTo`](https://community.bistudio.com/wiki/isEqualTo) not [`==`](https://community.bistudio.com/wiki/a%20==%20b).